stubs/openpyxl/openpyxl/xml/_functions.pyi

```diff
@@ -0,0 +1,60 @@
+# This file does not exist at runtime. It is a helper file to overload imports in openpyxl.xml.functions
+# TODO: Create missing stubs and add missing types upstream
```

Comment:
eh, maybe not. lxml is fine (as it's what this library was originally built on), but the other two sounds overkill.